### PR TITLE
Fixed base advertisment script so stashcp is correctly advertised. Ad…

### PR DESCRIPTION
**Pull request by rynge**
…ded missing userenv ad cron job.

### CODE REVIEW

File: 10-setup-htcondor.sh [hunk 1]
```diff
@@ -346,6 +348,23 @@ MASTER_ATTRS = \$(MASTER_ATTRS) AnnexName ACCEPT_JOBS_FOR_HOURS ACCEPT_IDLE_MINU
...
+STARTD_CRON_JOBLIST = $(STARTD_CRON_JOBLIST) base userenv
+
+STARTD_CRON_base_EXECUTABLE = /usr/sbin/osgvo-advertise-base
+STARTD_CRON_base_PERIOD = 4m
+STARTD_CRON_base_MODE = periodic
+STARTD_CRON_base_RECONFIG = true
+STARTD_CRON_base_KILL = true
+STARTD_CRON_base_ARGS =
+
+STARTD_CRON_userenv_EXECUTABLE = $LOCAL_DIR/main/singularity_wrapper.sh
```
Comment (matyasselmeci) [inline 10-setup-htcondor.sh:361]:
```suggestion
STARTD_CRON_userenv_EXECUTABLE = $LOCAL_DIR/main/osgvo-advertise-userenv
```
no?

Comment (rynge) [reply 10-setup-htcondor.sh:361]: No, it has to run via the singularity wrapper. The userenv is adverting the environment inside the default image.

Comment (matyasselmeci) [reply 10-setup-htcondor.sh:361]: I see.

File: 10-setup-htcondor.sh [hunk 2]
```diff
@@ -346,6 +348,23 @@ MASTER_ATTRS = \$(MASTER_ATTRS) AnnexName ACCEPT_JOBS_FOR_HOURS ACCEPT_IDLE_MINU
...
+STARTD_CRON_userenv_PERIOD = 4m
+STARTD_CRON_userenv_MODE = periodic
+STARTD_CRON_userenv_RECONFIG = true
+STARTD_CRON_userenv_KILL = true
+STARTD_CRON_userenv_ARGS = /usr/sbin/osgvo-advertise-base $LOCAL_DIR/glidein_config main
```
Comment (matyasselmeci) [inline 10-setup-htcondor.sh:366]:
```suggestion
STARTD_CRON_userenv_ARGS = /usr/sbin/osgvo-advertise-userenv $LOCAL_DIR/glidein_config main
```

**Review by matyasselmeci — APPROVED** [review]

**Pull request merged**